feature-group-definitions/async-await.yml

```
@@ -1,4 +1,5 @@
 name: Async functions
+description: The `async` and `await` keywords enable the asynchronous, promise-based behavior of a function to be written without using promise chains.
```

What do you think of saying what this means in a more explicit style, like this?

```suggestion
description: The `async` and `await` keywords allow you to write asynchronous functions without using using the `Promise()` constructor and chains of `then()` methods.
```

---

I think this is perhaps a bit too explicit. Async/await is also useful when you use other APIs that return promises, even if there's just one of them. So it's not necessarily about the constructor or a chain.

---

OK, in that case, rewriting solely to avoid "enables".

```suggestion
description: The `async` and `await` keywords allow you to use the asynchronous, promise-based behavior of a function without using promise chains.
```